cypress/support/pages/settings.email.pageObject.js

```diff
@@ -0,0 +1,9 @@
+class SettingsEmail {
+  updateEmail(newEmail) {
+    // eslint-disable-next-line cypress/unsafe-to-chain-command
+    cy.get('[placeholder="Email"]').clear().type(newEmail);
```

Comment:
Consider using `data-cy` attributes for selecting elements in your tests as per the task requirements. This will make your tests more robust and less prone to breaking if the UI changes.